dipy/denoise/localpca.py

```diff
@@ -10,39 +10,86 @@
 from scipy.linalg import eigh
 
 
-def localpca(arr, sigma, mask=None, pca_method='eig', patch_radius=2,
-             tau_factor=2.3, out_dtype=None):
+def _pca_classifier(L, m):
+    """ Classifies which PCA eigenvalues are related to noise and estimates the
+    noise variance
+
+    Parameters
```

Comment:
I guess `m` needs to be documented here...

Reply:
Done